src/main/java/Duke.java

```diff
@@ -1,10 +1,36 @@
+import java.io.IOException;
+import java.util.Scanner;
+
 public class Duke {
+    // todo should duke be instantiated?
+    private static TaskList taskList;
+
+    /**
+     * Entry point of the duke programme
+     * @param args Irrelevant argument
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner sc = new Scanner(System.in);
+
+        try {
+            taskList = TaskList.setupTaskList();
+        } catch (IOException e) {
+            Ui.print(new String[]{"Something went wrong in loading the task file and parsing",
+                    e.getMessage()});
+        }
+
+        Ui.intro();
+
+        // variables to reuse
+        String userInput;
+        Parser parser = new Parser(taskList);
+        boolean remainOpen = true;
```

Comment:
Perhaps this variable name could sound more boolean?

Reply:
Thanks for your review, all your comments were helpful. Will be adjusting my code accordingly soon.